CHANGELOG.md

```diff
@@ -33,6 +33,10 @@
   [keith](https://github.com/keith)
   [5139](https://github.com/realm/SwiftLint/pull/5139)
 
+* Tests now highlight disable and superfluous failures inline in example code  
+  [mredig](https://github.com/mredig)
+  [5149](https://github.com/realm/SwiftLint/pull/5149)
```

Comment:
An entry in the CHANGELOG is not necessary as this is nothing user-visible.

Reply:
Ah - sounds good. I'll get that updated soon.

Reply:
d u n Done!